manifests/rules/docker_ce.pp

```diff
@@ -80,44 +80,44 @@
   if $manage_docker_chains {
     nftables::chain {
       'DOCKER-nat':
-        table => 'ip-nat',
+        table => "ip-${nftables::nat_table_name}",
         chain => 'DOCKER';
     }
   }
 
   if $manage_base_chains {
     nftables::chain {
       'OUTPUT-nat':
-        table => 'ip-nat',
+        table => "ip-${nftables::nat_table_name}",
         chain => 'OUTPUT';
       'INPUT-nat':
-        table => 'ip-nat',
+        table => "ip-${nftables::nat_table_name}",
         chain => 'INPUT';
     }
   }
 
   nftables::rule {
     'POSTROUTING-docker':
-      table   => 'ip-nat',
+      table   => "ip-${nftables::nat_table_name}",
       content => "oifname != \"${docker_interface}\" ip saddr ${docker_prefix} counter masquerade";
     'PREROUTING-docker':
-      table   => 'ip-nat',
+      table   => "ip-${nftables::nat_table_name}",
       content => 'fib daddr type local counter jump DOCKER';
     'OUTPUT-jump_docker@ip-nat':
```

Comment:
I'd suggest to make the resource name match the table name using the variable (several cases in this file).